### PR TITLE
readme: Fix configuration for xsession

### PR DESCRIPTION
**Pull request by DamienCassou**
### Description

Improve the README so it is clear how to let home-manager configure the X session.

If you want, I can move this section to the FAQ and/or to the list of options.

### Checklist

- [x] Commit messages are formatted

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -284,7 +284,19 @@ like
 {
   # …
 
-  services.xserver.enable = true;
+  services.xserver = {
+    enable = true;
+
+    desktopManager.session = [
```
Comment (NickHu) [inline README.md:290]: I think you meant `displayManager.session`

File: README.md [hunk 2]
```diff
@@ -296,8 +308,13 @@ in your system configuration and
 {
   # …
 
-  xsession.enable = true;
-  xsession.windowManager.command = "…";
+  xsession = {
+    enable = true;
+    scriptPath = ".hm-xsession";
```
Comment (NickHu) [inline README.md:313]: I don't think it's necessary to change this from the default `~/.xsession`

**Review by NickHu — CHANGES_REQUESTED** [review]
Thanks for the contribution! I have this in my configs too, but I can't really remember why... for the sake of posterity, would you mind writing up what exactly this does as opposed to what's currently in the README?

**Comment by DamienCassou** [comment]
Thank you @NickHu for your review. I'm not sure what this PR is about anymore and I changed my home-manager configuration quite a bit since then. I'm closing this issue.

**Pull request closed without merging**